### PR TITLE
Moved contentfrom PCAxis.Encryption into PCAxis.Sql

### PR DESCRIPTION
**Pull request by likp**
Moved PCAxis.Encryption into PCAxis.Sql and removed the dependency to PCAxis.Encryption nuget in PCAxis.Sql

See https://github.com/statisticssweden/PxWeb/issues/163

### CODE REVIEW

File: PCAxis.Sql/PCAxis.Sql.csproj [hunk 1]
```diff
@@ -46,8 +46,8 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
     <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="2.19.110" />
-    <PackageReference Include="PCAxis.Core" Version="1.2.2" />
-    <PackageReference Include="PCAxis.Encryption" Version="1.0.2" />
+    <PackageReference Include="PCAxis.Core" Version="1.1.0" />
+    <PackageReference Include="System.Security.Cryptography.Xml" Version="7.0.1" />
```
Comment (JohannesFinsveen) [inline PCAxis.Sql/PCAxis.Sql.csproj:50]: PCAxis.Core downgrade on purpose?

Comment (likp) [reply PCAxis.Sql/PCAxis.Sql.csproj:50]: No looks like a misstake. I corrected it https://github.com/statisticssweden/PCAxis.Sql/blob/dfe28e0000a8dc368d2fcdadd7b1b25ea97ef753/PCAxis.Sql/PCAxis.Sql.csproj#L49 the PR still looks like it is still 1.1.0.

**Review by JohannesFinsveen — APPROVED** [review]
👍 very good

**Comment by likp** [comment]
fixes https://github.com/statisticssweden/PxWeb/issues/381

**Pull request merged**